changelogs/fragments/5034-bitwarden-module.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - bitwarden - added bitwarden support.
```

Comment:
New modules should not have changelog fragments. Please remove this file.

Reply:
This still needs to be implemented.

Reply:
Thanks for checking! I thought I had done this through the GitHub UI. I should stick to the command-line :/